### DISCUSSION TITLE
Unable to use MDAnalysis on my LAMMPS datafile and coordinate dump file

### DISCUSSION THREAD

**Discussion by wisecashew** [MDAnalysis/mdanalysis#4715] [General]
I am running a single polymer chain in water simulation. I am trying to create the Universe object for my simulation. My datafile is "pnipam_tip4p-ice-water.data" and my trajectory file is "coords.lammpstrj". It is a standard LAMMPS dump file with "atom" style format. 
`dump crds all atom 10000 coords.lammpstrj`. 

I am trying to create the Universe object the following way:
```
import MDAnalysis as mda 

if __name__=="__main__":

    dfile = "pnipam_tip4p-ice-water.r1.data"
    cfile = "coords.lammpstrj"

    u = mda.Universe(dfile, cfile, topology_format="LAMMPSDATA", format="LAMMPSDUMP", n_atoms=60572)
    # Access and print bonds
    print("Bonds:")
    for bond in u.bonds:
        print(f"Bond between atom {bond.atoms[0].id} and atom {bond.atoms[1].id}")
```
However, I get the output/error:
```
Bonds:
Traceback (most recent call last):
  File "/scratch/gpfs/satyend/MC_POLYMER/polymer_lattice/lattice_md/py_analysis/LAMMPS/MDA_reader.py", line 15, in <module>
    for bond in u.bonds:
  File "/home/satyend/.conda/envs/phase/lib/python3.9/site-packages/MDAnalysis/core/universe.py", line 666, in bonds
    return self.atoms.bonds
  File "/home/satyend/.conda/envs/phase/lib/python3.9/site-packages/MDAnalysis/core/groups.py", line 2537, in __getattr__
    return super(AtomGroup, self).__getattr__(attr)
  File "/home/satyend/.conda/envs/phase/lib/python3.9/site-packages/MDAnalysis/core/groups.py", line 609, in __getattr__
    raise NoDataError(err.format(singular=cls.singular))
MDAnalysis.exceptions.NoDataError: This Universe does not contain bonds information
```
How can I resolve this issue? Why isn't the Universe object holding all the bonds? I would appreciate any advice you have for me!

I have attached my data file (pnipam_tip4p-ice-water.data.txt) and a timestep (example.lammpstrj.txt) in this message. The .txt extension has been added to make sure such a file can be pasted on this forum. 

[example.lammpstrj.txt](https://github.com/user-attachments/files/17092207/example.lammpstrj.txt)
[pnipam_tip4p-ice-water.data.txt](https://github.com/user-attachments/files/17092209/pnipam_tip4p-ice-water.data.txt)

**Comment by orbeckst** [comment]
With your example files, I can read everything
```python
u = mda.Universe("pnipam_tip4p-ice-water.data", "example.lammpstrj", format="LAMMPSDUMP")
```
without having to give `n_atoms` or the topology format (which may be forcing MDAnalysis to choose an incorrect parser... not sure).
```python
In [23]: u.atoms.bonds
Out[23]: <TopologyGroup containing 40571 bonds>

In [24]: u.trajectory
Out[24]: <DumpReader example.lammpstrj with 1 frames of 60572 atoms>
```
